models/place-terms.json

```diff
@@ -4,6 +4,15 @@
   "description": "A list of taxonomy terms used to categorize the [place](./place.json).\n\nTerms are pre-defined and can be browsed in our [Terms](../docs/terms.md) guide.\n\nA place requires at least one term, to define its type. (For example a museum, monument, ...)",
   "uniqueItems": true,
   "minItems": 1,
+  "examples": [
+    [
+      {
+        "id": "0.50.4.0.0",
+        "label": "Concert",
+        "domain": "eventtype"
+      }
```

Comment:
Nit: Maybe us an actual `eventtype` that is valid from places as an example here. https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/type_locatie.html

Which ones are valid for events and which ones for places will become clearer for API integrators in these new docs once the new taxonomy JSON API is done.